<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Participant</class>
 <widget class="QWidget" name="Participant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Participant</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Answer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="answerEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="answerButton">
     <property name="text">
      <string>Answer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="scoreLabel">
     <property name="text">
      <string>Score: </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
